api/users/serializers.py

```diff
@@ -252,3 +252,17 @@ def get_absolute_url(self, obj):
 
     class Meta:
         type_ = 'institutions'
+
+
+class UserIdentitiesSerializer(BaseAPISerializer):
+    data = ser.DictField()
+    links = LinksField({'self': 'get_self_url'})
+
+    def get_self_url(self, obj):
```

Comment:
The `self` url we want to return here should be the detail url, not the list url.